### ISSUE TITLE
Pylance complains "Literal[10]" is incompatible with "property" when using modules as implementations of protocol

### ISSUE THREAD

**Issue by rk-terence** [microsoft/pyright#6117]
## Environment data

-   Language Server version: v2023.10.10
-   OS and version: macOS Sonoma
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.10.13 (main, Aug 24 2023, 22:36:46) [Clang 14.0.3 (clang-1403.0.22.14.1)] on darwin

## Code Snippet

```python
# main.py
import abc
import typing

import example


class PersonProto(typing.Protocol):
    @property
    @abc.abstractmethod
    def AGE(self) -> int: ...


class Person:
    AGE = 10


p1: PersonProto = Person()  # This line passes checking
p2: PersonProto = example  # This line does not pass checking
```

Inside `example.py`:
```python
# example.py
AGE = 10
```

## Expected behavior

I expect that Pylance does not complain this.

## Actual behavior

When hovering on the `example` variable, Pylance shows the detailed complaints:

```
Expression of type "Module("example")" cannot be assigned to declared type "PersonProto"
  "AGE" is an incompatible type
    "Literal[10]" is incompatible with "property"Pylance[reportGeneralTypeIssues](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportGeneralTypeIssues)
```

Screenshot:
<img width="617" alt="image" src="https://github.com/microsoft/pyright/assets/28799257/7b0ba717-b7f8-41bb-8e6d-13090fb571c8">

This is a strange behavior for me, as the types of `AGE` in both `example.py` and in `class Person` are `Literal[10]`, but one assignment is fine, while the other triggers the complaints of Pylance. Are there anything special with module objects?

## Logs

```
2023-10-11 13:01:32.571 [info] [Info  - 13:01:32] (37260) Pylance language server 2023.10.10 (pyright 45942a15) starting
2023-10-11 13:01:32.571 [info] [Info  - 13:01:32] (37260) Server root directory: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist
2023-10-11 13:01:32.571 [info] [Info  - 13:01:32] (37260) Starting service instance "mre-pylance-bug"
2023-10-11 13:01:32.587 [info] (37260) No configuration file found.
2023-10-11 13:01:32.587 [info] (37260) No pyproject.toml file found.
2023-10-11 13:01:32.587 [info] [Info  - 13:01:32] (37260) Setting pythonPath for service "mre-pylance-bug": "/opt/homebrew/bin/python3.10"
2023-10-11 13:01:32.587 [info] [Info  - 13:01:32] (37260) Setting environmentName for service "mre-pylance-bug": "3.10.13 (global)"
2023-10-11 13:01:32.587 [info] [Warn  - 13:01:32] (37260) stubPath /Users/gz/Documents/tmp/mre-pylance-bug/typings is not a valid directory.
2023-10-11 13:01:32.605 [info] [Info  - 13:01:32] (37260) Assuming Python version 3.10
2023-10-11 13:01:32.606 [info] (37260) Assuming Python platform Darwin
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260) Search paths for /Users/gz/Documents/tmp/mre-pylance-bug
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /Users/gz/Documents/tmp/mre-pylance-bug
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /Users/gz/Documents/tmp/mre-pylance-bug/typings
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stubs/...
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/bundled/stubs
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260)   /opt/homebrew/lib/python3.10/site-packages
2023-10-11 13:01:32.660 [info] [Info  - 13:01:32] (37260) Adding fs watcher for library directories:
 /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10
/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload
/opt/homebrew/lib/python3.10/site-packages
2023-10-11 13:01:32.661 [info] [Info  - 13:01:32] (37260) Adding fs watcher for directories:
 /Users/gz/Documents/tmp/mre-pylance-bug
2023-10-11 13:01:32.661 [info] (37260) Searching for source files
2023-10-11 13:01:32.662 [info] [Info  - 13:01:32] (37260) Found 2 source files
2023-10-11 13:01:32.885 [info] (37260) [FG] parsing: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (7ms)
2023-10-11 13:01:32.921 [info] (37260) [FG] parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/builtins.pyi [fs read 2ms] (35ms)
2023-10-11 13:01:32.940 [info] (37260) [FG] binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/builtins.pyi (18ms)
2023-10-11 13:01:32.941 [info] (37260) [FG] binding: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (0ms)
2023-10-11 13:01:32.945 [info] (37260) [BG(1)] getSemanticTokens full at /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py ...
2023-10-11 13:01:32.945 [info] (37260) [BG(1)]   parsing: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (68ms)
2023-10-11 13:01:32.969 [info] (37260) [IDX(2)] scan packages /Users/gz/Documents/tmp/mre-pylance-bug ...
2023-10-11 13:01:32.969 [info] (37260) [IDX(2)]   read stdlib indices (12ms)
2023-10-11 13:01:32.984 [info] (37260) [BG(1)]   parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/builtins.pyi [fs read 3ms] (38ms)
2023-10-11 13:01:33.002 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/builtins.pyi (18ms)
2023-10-11 13:01:33.002 [info] (37260) [BG(1)]   binding: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (0ms)
2023-10-11 13:01:33.015 [info] (37260) [BG(1)]   parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/typing.pyi [fs read 0ms] (9ms)
2023-10-11 13:01:33.019 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/typing.pyi (4ms)
2023-10-11 13:01:33.024 [info] (37260) [BG(1)]   parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/typing_extensions.pyi [fs read 0ms] (2ms)
2023-10-11 13:01:33.025 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/typing_extensions.pyi (1ms)
2023-10-11 13:01:33.036 [info] (37260) [BG(1)]   parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/_typeshed/__init__.pyi [fs read 0ms] (2ms)
2023-10-11 13:01:33.037 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/_typeshed/__init__.pyi (1ms)
2023-10-11 13:01:33.046 [info] (37260) [BG(1)]   parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/types.pyi [fs read 0ms] (8ms)
2023-10-11 13:01:33.048 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/types.pyi (1ms)
2023-10-11 13:01:33.051 [info] (37260) [BG(1)]   parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/abc.pyi [fs read 0ms] (1ms)
2023-10-11 13:01:33.051 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/abc.pyi (0ms)
2023-10-11 13:01:33.053 [info] (37260) [BG(1)]   parsing: /Users/gz/Documents/tmp/mre-pylance-bug/example.py [fs read 1ms] (1ms)
2023-10-11 13:01:33.053 [info] (37260) [BG(1)]   binding: /Users/gz/Documents/tmp/mre-pylance-bug/example.py (0ms)
2023-10-11 13:01:33.055 [info] (37260) [BG(1)]   parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/collections/abc.pyi [fs read 0ms] (0ms)
2023-10-11 13:01:33.056 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/collections/abc.pyi ...
2023-10-11 13:01:33.057 [info] (37260) [BG(1)]     parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/_collections_abc.pyi [fs read 0ms] (1ms)
2023-10-11 13:01:33.057 [info] (37260) [BG(1)]     binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/_collections_abc.pyi (0ms)
2023-10-11 13:01:33.057 [info] (37260) [BG(1)]   binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/collections/abc.pyi (1ms)
2023-10-11 13:01:33.065 [info] (37260) [BG(1)] getSemanticTokens full at /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (188ms)
2023-10-11 13:01:33.066 [info] (37260) [BG(1)] getSemanticTokens range 0:0 - 38:0 at /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (1ms)
2023-10-11 13:01:33.082 [info] (37260) [BG(1)] analyzing: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py ...
2023-10-11 13:01:33.083 [info] (37260) [BG(1)]   checking: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py ...
2023-10-11 13:01:33.083 [info] (37260) [BG(1)]     parsing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/abc.py [fs read 0ms] (4ms)
2023-10-11 13:01:33.083 [info] [Info  - 13:01:33] (37260) Could not import '_abc' in file '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/abc.py'
2023-10-11 13:01:33.083 [info] [Info  - 13:01:33] (37260)   Looking in stubPath '/Users/gz/Documents/tmp/mre-pylance-bug/typings'
2023-10-11 13:01:33.083 [info] [Info  - 13:01:33] (37260)   Attempting to resolve stub package using root path '/Users/gz/Documents/tmp/mre-pylance-bug/typings'
2023-10-11 13:01:33.084 [info] [Info  - 13:01:33] (37260)   Attempting to resolve using root path '/Users/gz/Documents/tmp/mre-pylance-bug/typings'
2023-10-11 13:01:33.084 [info] [Info  - 13:01:33] (37260)   Looking in root directory of execution environment '/Users/gz/Documents/tmp/mre-pylance-bug'
2023-10-11 13:01:33.084 [info] [Info  - 13:01:33] (37260)   Attempting to resolve stub package using root path '/Users/gz/Documents/tmp/mre-pylance-bug'
2023-10-11 13:01:33.085 [info] [Info  - 13:01:33] (37260)   Attempting to resolve using root path '/Users/gz/Documents/tmp/mre-pylance-bug'
2023-10-11 13:01:33.086 [info] [Info  - 13:01:33] (37260)   Finding python search paths
2023-10-11 13:01:33.086 [info] [Info  - 13:01:33] (37260)   Executing interpreter: '/opt/homebrew/bin/python3.10'
2023-10-11 13:01:33.086 [info] [Info  - 13:01:33] (37260)   Skipping '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python310.zip' because it is not a valid directory
2023-10-11 13:01:33.086 [info] [Info  - 13:01:33] (37260)   Received 3 paths from interpreter
2023-10-11 13:01:33.086 [info] [Info  - 13:01:33] (37260)     /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10
2023-10-11 13:01:33.086 [info] [Info  - 13:01:33] (37260)     /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload
2023-10-11 13:01:33.086 [info] [Info  - 13:01:33] (37260)     /opt/homebrew/lib/python3.10/site-packages
2023-10-11 13:01:33.087 [info] [Info  - 13:01:33] (37260)   Looking in python search path '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10'
2023-10-11 13:01:33.087 [info] [Info  - 13:01:33] (37260)   Attempting to resolve stub package using root path '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10'
2023-10-11 13:01:33.087 [info] [Info  - 13:01:33] (37260)   Attempting to resolve using root path '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10'
2023-10-11 13:01:33.087 [info] [Info  - 13:01:33] (37260)   Looking in python search path '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload'
2023-10-11 13:01:33.088 [info] [Info  - 13:01:33] (37260)   Attempting to resolve stub package using root path '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload'
2023-10-11 13:01:33.088 [info] [Info  - 13:01:33] (37260)   Attempting to resolve using root path '/opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload'
2023-10-11 13:01:33.089 [info] [Info  - 13:01:33] (37260)   Looking in python search path '/opt/homebrew/lib/python3.10/site-packages'
2023-10-11 13:01:33.089 [info] [Info  - 13:01:33] (37260)   Attempting to resolve stub package using root path '/opt/homebrew/lib/python3.10/site-packages'
2023-10-11 13:01:33.091 [info] [Info  - 13:01:33] (37260)   Attempting to resolve using root path '/opt/homebrew/lib/python3.10/site-packages'
2023-10-11 13:01:33.091 [info] [Info  - 13:01:33] (37260)   Looking in bundled stubs path '/Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/bundled/stubs'
2023-10-11 13:01:33.092 [info] [Info  - 13:01:33] (37260)   Attempting to resolve stub package using root path '/Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/bundled/stubs'
2023-10-11 13:01:33.093 [info] [Info  - 13:01:33] (37260)   Attempting to resolve using root path '/Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/bundled/stubs'
2023-10-11 13:01:33.094 [info] [Info  - 13:01:33] (37260)   Looking for typeshed stdlib path
2023-10-11 13:01:33.094 [info] [Info  - 13:01:33] (37260)   Looking for typeshed stdlib path
2023-10-11 13:01:33.095 [info] [Info  - 13:01:33] (37260)   Attempting to resolve using root path '/Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib'
2023-10-11 13:01:33.095 [info] [Info  - 13:01:33] (37260)   Typeshed path not found
2023-10-11 13:01:33.096 [info] [Info  - 13:01:33] (37260)   Looking for typeshed third-party path
2023-10-11 13:01:33.096 [info] [Info  - 13:01:33] (37260)   Looking for typeshed stubs path
2023-10-11 13:01:33.097 [info] [Info  - 13:01:33] (37260)   Typeshed path not found
2023-10-11 13:01:33.097 [info] (37260) [BG(1)]     binding: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/abc.py (2ms)
2023-10-11 13:01:33.112 [info] (37260) [BG(1)]     parsing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/typing.py [fs read 1ms] (28ms)
2023-10-11 13:01:33.119 [info] (37260) [BG(1)]     binding: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/typing.py (7ms)
2023-10-11 13:01:33.127 [info] (37260) [BG(1)]   checking: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (60ms)
2023-10-11 13:01:33.127 [info] (37260) [BG(1)] analyzing: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py (61ms)
2023-10-11 13:01:33.129 [info] (37260) [BG(1)] indexing: /Users/gz/Documents/tmp/mre-pylance-bug/example.py [found 1] (0ms)
2023-10-11 13:01:33.130 [info] (37260) [BG(1)] indexing: /Users/gz/Documents/tmp/mre-pylance-bug/protocol_bug.py [found 4] (0ms)
2023-10-11 13:01:33.263 [info] (37260) [IDX(2)] scan packages /Users/gz/Documents/tmp/mre-pylance-bug (304ms)
2023-10-11 13:01:33.282 [info] (37260) [IDX(2)] index packages /Users/gz/Documents/tmp/mre-pylance-bug ...
2023-10-11 13:01:33.282 [info] (37260) [IDX(2)]   index execution environment /Users/gz/Documents/tmp/mre-pylance-bug ...
2023-10-11 13:01:33.282 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/concurrent/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.282 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/distutils/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.282 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/idlelib/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.282 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib2to3/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.283 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/pydoc_data/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.283 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/test/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.283 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/turtledemo/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.283 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.283 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/wsgiref/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.327 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/xml/__init__.py ...
2023-10-11 13:01:33.327 [info] (37260) [IDX(2)]       parsing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/xml/__init__.py [fs read 0ms] (44ms)
2023-10-11 13:01:33.353 [info] (37260) [IDX(2)]       parsing: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/builtins.pyi [fs read 1ms] (26ms)
2023-10-11 13:01:33.360 [info] (37260) [IDX(2)]       binding: /Users/gz/.vscode/extensions/ms-python.vscode-pylance-2023.10.10/dist/typeshed-fallback/stdlib/builtins.pyi (7ms)
2023-10-11 13:01:33.360 [info] (37260) [IDX(2)]       binding: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/xml/__init__.py (0ms)
2023-10-11 13:01:33.360 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/xml/__init__.py [found 0] (78ms)
2023-10-11 13:01:33.360 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.13/Frameworks/Python.framework/Versions/3.10/lib/python3.10/xmlrpc/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.361 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pip/__init__.py ...
2023-10-11 13:01:33.361 [info] (37260) [IDX(2)]       parsing: /opt/homebrew/lib/python3.10/site-packages/pip/__init__.py [fs read 0ms] (1ms)
2023-10-11 13:01:33.361 [info] (37260) [IDX(2)]       binding: /opt/homebrew/lib/python3.10/site-packages/pip/__init__.py (0ms)
2023-10-11 13:01:33.361 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pip/__init__.py [found 2] (1ms)
2023-10-11 13:01:33.362 [info] (37260) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/wheel/__init__.py [skipped: no '__all__' defined] (0ms)
2023-10-11 13:01:33.362 [info] (37260) [IDX(2)]   index execution environment /Users/gz/Documents/tmp/mre-pylance-bug [found 98 in 4 files] (82ms)
2023-10-11 13:01:33.362 [info] (37260) [IDX(2)] index packages /Users/gz/Documents/tmp/mre-pylance-bug [found 98 in 1 exec envs] (84ms)
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by rk-terence** [comment]
Thank you for your fast response!

**Comment by erictraut** [comment]
This is included in pyright 1.1.332, which I just published. It will also be included in a future release of pylance.
